drivers/watchdog/wdt_nrfx.c

```diff
@@ -166,6 +167,7 @@ static const struct wdt_driver_api wdt_nrfx_driver_api = {
 	.feed = wdt_nrf_feed,
 };
 
+#if !defined(CONFIG_WDT_NRFX_NO_IRQ) || (CONFIG_WDT_NRFX_NO_IRQ == 0)
```

Comment:
why is `== 0` here? Kconfigs are either defined or not

Reply:
Removed CONFIG_WDT_NRFX_NO_IRQ == 0